src/Symfony/Component/Security/Guard/GuardAuthenticatorInterface.php

```diff
@@ -39,11 +39,16 @@
      * Whatever value you return here will be passed to getUser() and checkCredentials()
      *
      * For example, for a form login, you might:
-     *
-     *      return array(
-     *          'username' => $request->request->get('_username'),
-     *          'password' => $request->request->get('_password'),
-     *      );
+     * 
+     *      if($request->request->has('_username')) {
```

Comment:
Sorry again for the CS, I should really forget my old habits and get used to real PSR-2...

For the `else` I want to emphasis on the fact that you should not forget to return null when you don't have credentials.